src/main/java/com/ericsson/ei/status/StatusData.java

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.status;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class StatusData {
+
+    private Status eiffelIntelligenceStatus = Status.NOT_SET;
```

Comment:
Isn't this same thing as UNAVAILABLE/NOT_OK state. Default can be Unvailable until when connections is up and state changes to Available/OK state.

Reply:
UNAVAILABLE means unavailable, NOT_SET means it has yet to be updated, there is a logical difference and we decided to keep it like this.

Reply:
If it is not set then its also unavailable until the check says it is available.
So I don't see any good motivation to keep the extra not_set state, since it can only be used in the initiation phase of application.

Reply:
No, if NOT_SET the service may or may not work, UNAVAILABLE means it does not work.
NOT_SET: Service may (not) work, use on your own risk.
UNAVAILABLE: Service will not work, do not use.
AVAILABLE: Se guarantee that the service works.

Reply:
Ok, where are you using NOT_SET in other places? I really don't see the use case to have this extra state, since it will change directly to unavailable or available nad NOT_SET will never be used. Which means we could set it directly to unavailable.
But keep it as is then, if you other agree with Anders opinion.